src/components/SchemaFormInput.vue

```diff
@@ -50,6 +50,13 @@
   grid
   gap-1
 }
+
+.schema-form-input__component { @apply
+  max-w-full
+  min-h-[2.5rem]
+  resize-y
+}
```

Comment:
This seems a little too specific to apply to all schema components. The max width makes sense but the min-h and resize?

Reply:
I can target the code inputs instead, wdyt?

Reply:
I'm not sure, this just seems overly generic for this. Could be wrong 🤷

Reply:
I added a more targeted selector for now cause I agree that was too generic. That'll let us close some tickets around resizing these inputs too